src/App.js

```diff
@@ -1,13 +1,119 @@
 import React from 'react';
 import './App.css';
+import Product from "./components/Products";
+import Button from "./components/Button";
+import Tile from "./components/Tile";
+
+import bag_1 from "./assets/bag_1.png";
+import bag_2 from "./assets/bag_2.png";
+import bag_3 from "./assets/bag_3.png";
+import bag_4 from "./assets/bag_4.png";
+import brand from "./assets/brand.png";
+import story from "./assets/our_story.png";
+
+
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
 
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+
+            <nav>
+
+                <Button
+                    text="To the collection"
```

Comment:
In de uitwerkingen van het voorbeeld is er bij elke button nog een clickHandler toegevoegd, zodat er in de console bijvoorbeeld gelogd wordt: "To the collection" wanneer iemand op de button klikt.
bijv. clickHandler={() => console.log("To the collection!")}

Reply:
Oh wacht, dit heb je in het button component zelf gemaakt zie ik nu, negeer bovenstaande comment maar 😆